libview/ev-view.c

```diff
@@ -2294,7 +2294,7 @@ ev_view_form_field_choice_changed (GtkWidget   *widget,
 {
 	EvFormFieldChoice *field_choice = EV_FORM_FIELD_CHOICE (field);
 	
-	if (gtk_combo_box_get_has_entry ( GTK_COMBO_BOX (widget))) {
```

Comment:
It makes no sense to check this here and on line 2300 again, which is inside the if-block itself.